CMakeLists.txt

```diff
@@ -20,9 +20,11 @@ project(rdycore)
 set(PETSC_DIR $ENV{PETSC_DIR})
 set(PETSC_ARCH $ENV{PETSC_ARCH})
 find_package(PETSc COMPONENTS C)
+list(APPEND CMAKE_PREFIX_PATH ${PETSC_DIR}/${PETSC_ARCH}/lib/pkgconfig)
 
 include(FindPkgConfig)
 pkg_check_modules(CEED REQUIRED IMPORTED_TARGET ceed)
+include_directories(${CEED_INCLUDE_DIRS})
```

Comment:
This line already exists down below next to this:
```
include_directories(${PETSC_INCLUDES})
```
Can we delete one or the other?

Reply:
Sure, I'll make sure we don't need the `CEED_INCLUDE_DIRS`. I'm having some issues getting the CI environment to work with this PR, but I'll prune this when I've figured it out.